api/validation/schemas.py

```diff
@@ -0,0 +1,26 @@
+from marshmallow import Schema, fields, validate, INCLUDE
+
+from api.validation.validators import comma_splittable, aws_region_validator
+
+
+class EC2ActionSchema(Schema):
+    action = fields.String(required=True, validate=validate.OneOf(['stop_instances', 'start_instances']))
```

Comment:
question: what's the correct tradeoff between typing a field as string, or enumerate all possible values? In this case the parameter has only two values, so it's fairly easy to do a precise typing, but what about an endpoint which allows, for example, 10 different actions? Do we have to enumerate them all? What's your recommendation?

Reply:
My personal recommendation is to be as strict as possible, when it makes sense (so without getting crazy).
For example for PC available regions, we know that values will rarely change, so we can handle them all.

I would keep having this approach if values don't change too often and are not too big of a number, and they're static values.

For all other instances we can just type a field as string and let the target service do the rest